configs/ray-data-batch-classification/aws.yaml

```diff
@@ -0,0 +1,5 @@
+head_node_type:
+  name: head
+  instance_type: m5.2xlarge
+worker_node_types: []
+auto_select_worker_config: true
```

Comment:
not sure when this got merged, but this should be just using the default serverless config instead of defining a new config - @scottsun94